src/main/java/erb/ui/ScheduleAddDialog.java

```diff
@@ -52,10 +60,10 @@
     JLabel lblBus = new JLabel();
     JLabel lblDriver = new JLabel();
     JLabel lblRoute = new JLabel();
-    public static JComboBox addScheduleTimeField = new JComboBox();
-    public static JComboBox addScheduleBusField = new JComboBox();
-    public static JComboBox addScheduleDriverField = new JComboBox();
-    public static JComboBox addScheduleRouteField = new JComboBox();
+    public static final JComboBox addScheduleTimeField = new JComboBox();
```

Comment:
All final should have underscore and be capitalized  _ADD_SCHEDULE_TIME_FIELD
```suggestion
    public static final JComboBox _ADD_SCHEDULE_TIME_FIELD = new JComboBox();
```

Reply:
good catch